routing/control_tower_test.go

```diff
@@ -182,6 +182,137 @@ func TestPaymentControlSubscribeFail(t *testing.T) {
 	})
 }
 
+func TestPaymentControlSubscribeAllSuccess(t *testing.T) {
+	t.Parallel()
+
+	db, err := initDB()
+	if err != nil {
```

Comment:
nit: new code should use the `require` library.

Reply:
Done, I hope I did this in a proper way